mesonbuild/compilers/compilers.py

```diff
@@ -1048,10 +1048,16 @@ def get_cross_extra_flags(self, environment, link):
             if 'properties' in environment.cross_info.config:
                 props = environment.cross_info.config['properties']
                 lang_args_key = self.language + '_args'
-                extra_flags += props.get(lang_args_key, [])
+                lang_args = props.get(lang_args_key, [])
+                if isinstance(lang_args, str):
```

Comment:
Can't you use stringlistify here?

Reply:
I forgot we had that. :man_facepalming: Updating.